packages/backend/src/graphql/schema.graphql

```diff
@@ -479,6 +496,21 @@ input CreateAccountInput {
   publicName: String
 }
 
+type Client implements Model {
```

Comment:
One of the required fields in auth's `ClientDetails` type that isn't present here is `email`.

Reply:
And `image`. Will update it 👍

Reply:
This has been fixed.